seed/data_importer/tasks.py

```diff
@@ -275,7 +308,7 @@ def map_row_chunk(ids, file_pk, source_type, prog_key, increment, **kwargs):
         table_mappings['PropertyState'] = debug_inferred_prop_state_mapping
     # TODO: *END TOTAL TERRIBLE HACK**
 
-    map_cleaner = _build_cleaner(org)
+    map_cleaner = _build_cleaner_2(org)
```

Comment:
i assume we deprecate _build_cleaner when we move over to _build_cleaner_2? Does _build_cleaner exist anymore? If so, can you mark it as to be deprecated?

Reply:
maybe I should just replace the body of `_build_cleaner` for a clean diff, but I didn't want to do that without checking with you if that old units code was truly dead.